api/app/database_functions/utils/sessions_util.py

```diff
@@ -21,3 +25,6 @@ def create_session(user_id, role):
     db.execute_sql_write("INSERT INTO sessions (session_id, user_id, role, creation_time, expiration_time) VALUES (%s, %s, %s, %s, %s)",
                          params=(session_id, user_id, role, str(creation_time), str(expiration_time)))
     return session_id
+
+def is_logged_in(session_id):
+    return db.execute_sql_read_fetchone("SELECT COUNT(*) FROM sessions WHERE session_id = %s", params=(session_id,))[0] > 0
```

Comment:
will have to check if session is expired too, can somewhat refer to `get_session`

Reply:
sry never mind ignore this. expired session should be removed eventually (not sure the mechanism yet)